packages/form-utils/src/lib/Switch.tsx

```diff
@@ -12,6 +17,29 @@ export type SwitchProps = Omit<
   | 'touchRippleRef'
 >;
 
-export const Switch = (args: SwitchProps): JSX.Element => {
+const CheckedIconSwitch = styled(MuiSwitch)(({ theme }) => ({
```

Comment:
do we also want to fix this for legacy theme?

Reply:
Oh, yeah, good call. I'll fix it.

Reply:
Fixed